install.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import re
+import os
+import subprocess
+import shutil
+import argparse
+
+
+CHART = './fpga.chart.py'
+CONF = './fpga.conf'
+
+
+def get_flavour():
+    flavour_line = subprocess.check_output('cat /etc/os-release | grep -E \'^NAME="\'', shell=True)
+    flavour = re.match(r'^NAME=\"(\w+)', flavour_line.decode('UTF-8'))
+
+    return flavour.group(1)
+
+
+def check_for_package(cmd):
+    try:
+        DEVNULL = open(os.devnull, 'w')
+        out = subprocess.check_output(cmd + ' | grep netdata', stderr=DEVNULL, shell=True)
+        DEVNULL.close()
+        return True
+    except subprocess.CalledProcessError as exc:
+        return False
+
+def netdata_is_installed(linux_flavour):
```

Comment:
This won't work when the user installed it via the one-line bash installer, or?

Reply:
I think the best possible and also OS-agnostic version is to first check whether `/usr/sbin/netdata` exists and if not, throw an error. Additionally, the user could get an argument to force installation.

Reply:
There is also `shutil.which()`

Reply:
In centos the netdata binary is in /sbin